src/main/java/io/vertx/cassandra/CassandraClient.java

```diff
@@ -136,7 +136,7 @@ static CassandraClient createShared(Vertx vertx, String datasourceName, Cassandr
    * @param query         the query to execute
    * @return current Cassandra client instance
    */
-  @GenIgnore
+  @SuppressWarnings("codegen-allow-any-java-type")
```

Comment:
Can this annotation be used at class level?

Reply:
Nope